src/applications/vaos/tests/new-appointment/components/DateTimeRequestPage.unit.spec.jsx

```diff
@@ -9,7 +9,7 @@ import { FETCH_STATUS } from '../../../utils/constants';
 import { waitFor } from '@testing-library/dom';
 import { Route } from 'react-router-dom';
 
-describe('VAOS <DateTimeRequestPage>', () => {
```

Comment:
Should this be part of this PR?

Reply:
No, I was attempting to avoid the broken tests